.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```
@@ -0,0 +1,86 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
```

```suggestion
        Thank you for taking the time to file a bug report!
```

Let's tweak this it's more in the active voice.

---

Sounds good! I'll update also the issue forms via <https://github.com/stylelint/.github/pull/8>.